charts/rollout-operator/README.md

```diff
@@ -72,3 +72,7 @@ It is not a highly available application and runs as a single pod.
 | serviceMonitor.relabelings | list | `[]` | ServiceMonitor relabel configs to apply to samples before scraping https://github.com/prometheus-operator/prometheus-operator/blob/master/Documentation/api.md#relabelconfig |
 | serviceMonitor.scrapeTimeout | string | `nil` | ServiceMonitor scrape timeout in Go duration format (e.g. 15s) |
 | tolerations | list | `[]` |  |
+| webhooks.enabled | bool | `true` | Enable the rollout-operator webhooks. See https://github.com/grafana/rollout-operator/#webhooks |
+| webhooks.failurePolicy | string | `""` | Validating and mutating webhook failure policy. Ignore | Fail. Use Ignore when first installing and then re-apply with Fail for ongoing use |
+| webhooks.selfSignedCertSecretName | string | `"certificate"` | Secret resource name for the TLS certificate to be used with the webhooks |
+| webhooks.zoneAwarePodDisruptionBudgets | string | `nil` | Enable zone aware pod disruption budgets. See https://github.com/grafana/rollout-operator/#zoneawarepoddisruptionbudget |
```

Comment:
I don't think we need this - charts that depend on this one can add their own ZPDBs.